nav.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>navbar</title>
+	<link href="https://fonts.googleapis.com/css?family=Engagement|Raleway" rel="stylesheet">
+	<link rel="stylesheet" type="text/css" href="nav.css">
+	<meta name="viewport" content="width=device-width, initial-scale=1">
+</head>
+<body>
+	<nav id="main-nav">
+		<ul class="buttons">
+			<li><a href="index.html">Home</a></li>
+			<li><a href="gallerydog.html">Gallery</a></li>
+			<li><a href="events.html">Events</a></li>
+			<li><a href="Contact.html">Contact</a></li>
```

Comment:
Why is the file uppercase? Be consistent with everything. None of the other files are uppercase.